nixos/doc/manual/from_md/release-notes/rl-2205.section.xml

```diff
@@ -564,6 +564,14 @@
           </listitem>
         </itemizedlist>
       </listitem>
+      <listitem>
+        <para>
+          The option <literal>services.snapserver.openFirewall</literal>
+          will no longer default to <literal>true</literal> starting
+          with NixOS 22.11. Enable it explicitly if you need to control
+          Snapserver remotely.
```

Comment:
Snapcast is a multiroom audio solution, It is based on streaming audio in a client-server fashion. This setting does not only relate to remote control, but to the core functionality of the snapserver - streaming audio.

Reply:
Sorry for the snapcast 101, didn't realize you were the author of the module.

Reply:
No offense taken. I opened this PR because of #149889 with which I do agree, even if it means that virtually everyone who wants to use this module will need one extra line in their config.

Reply:
https://github.com/NixOS/nixpkgs/blob/16f6057072e80c64591c09acddc096068015408a/nixos/modules/services/audio/snapserver.nix#L313-L316

The changelog description is not correct, given that also audio streaming is controlled by this option.

Reply:
@tobim Could you pleaseregenrated the changelog after rebase and while at it also update the message to be more precise. 

Other than that I think this is ready to merge.

Reply:
https://github.com/NixOS/nixpkgs/blob/16f6057072e80c64591c09acddc096068015408a/nixos/modules/services/audio/snapserver.nix#L313-L316
What if `openFirewall` is false? Looks like 2 ports are still opened?

Reply:
I updated the PR according to your suggestions.